Inputs/config.js

```diff
@@ -20,104 +20,134 @@ const accountSetting = {
   portfolioId: 0
 }
 
-const symbols = [
-  'USDC/USDT',
-  'ALGO/USDT',
-  "ETH/USDT",
-  "BNB/USDT",
-  "BTC/USDT",
-  "XRP/USDT",
-  "ADA/USDT",
-  "TRX/USDT",
-  "BAT/USDT",
-  "LINK/USDT",
-  "MANA/USDT",
-  "SOL/USDT",
-  "MATIC/USDT",
-  "THETA/USDT",
-  "HNT/USDT",
-  "DOT/USDT",
-  "YFI/USDT",
-  "LUNA/USDT",
-  "UNI/USDT",
-  "AVAX/USDT",
-  "AAVE/USDT",
-  "QNT/USDT",
-  "AXS/USDT",
-  "FIL/USDT",
-  "ETH/USDC",
-  "BNB/USDC",
-  "BTC/USDC",
-  "XRP/USDC",
-  "ADA/USDC",
-  "BAT/USDC",
-  "LINK/USDC",
-  "SOL/USDC",
-  "TRX/USDC"
+let symbols = [
+  // 'USDC/USDT',
+  // 'ALGO/USDT',
+  // "ETH/USDT",
+  // "BNB/USDT",
+  "BTC/BUSD"
+  // "XRP/USDT",
+  // "ADA/USDT",
+  // "TRX/USDT",
+  // "BAT/USDT",
+  // "LINK/USDT",
+  // "MANA/USDT",
+  // "SOL/USDT",
+  // "MATIC/USDT",
+  // "THETA/USDT",
+  // "HNT/USDT",
+  // "DOT/USDT",
+  // "YFI/USDT",
+  // "LUNA/USDT",
+  // "UNI/USDT",
+  // "AVAX/USDT",
+  // "AAVE/USDT",
+  // "QNT/USDT",
+  // "AXS/USDT",
+  // "FIL/USDT",
+  // "ETH/USDC",
+  // "BNB/USDC",
+  // "BTC/USDC",
+  // "XRP/USDC",
+  // "ADA/USDC",
+  // "BAT/USDC",
+  // "LINK/USDC",
+  // "SOL/USDC",
+  // "TRX/USDC"
 ]
 
-const symbolsForMarkPrice = [
-  'USDC/USDT',
-  'ALGO/USDT',
+let symbolsForMarkPrice = [
   "ETH/USDT",
-  "BNB/USDT",
-  "BTC/USDT",
-  "XRP/USDT",
-  "ADA/USDT",
-  "TRX/USDT",
-  "BAT/USDT",
-  "LINK/USDT",
-  "MANA/USDT",
-  "SOL/USDT",
-  "MATIC/USDT",
-  "THETA/USDT",
-  "HNT/USDT",
-  "DOT/USDT",
-  "YFI/USDT",
-  "LUNA/USDT",
-  "UNI/USDT",
-  "AVAX/USDT",
-  "AAVE/USDT",
-  "QNT/USDT",
-  "AXS/USDT",
-  "FIL/USDT"
+  // "BNB/USDT",
+  "BTC/USDT"
+  // 'USDC/USDT',
+  // 'ALGO/USDT',
+  // "ETH/USDT",
+  // "BNB/USDT",
+  // "BTC/USDT",
+  // "XRP/USDT",
+  // "ADA/USDT",
+  // "TRX/USDT",
+  // "BAT/USDT",
+  // "LINK/USDT",
+  // "MANA/USDT",
+  // "SOL/USDT",
+  // "MATIC/USDT",
+  // "THETA/USDT",
+  // "HNT/USDT",
+  // "DOT/USDT",
+  // "YFI/USDT",
+  // "LUNA/USDT",
+  // "UNI/USDT",
+  // "AVAX/USDT",
+  // "AAVE/USDT",
+  // "QNT/USDT",
+  // "AXS/USDT",
+  // "FIL/USDT"
 ]
 
-const symbolsForSettlementPrice = [
-  'USDC/USDT',
-  'ALGO/USDT',
+let symbolsForSettlementPrice = [
   "ETH/USDT",
-  "BNB/USDT",
-  "BTC/USDT",
-  "XRP/USDT",
-  "ADA/USDT",
-  "TRX/USDT",
-  "BAT/USDT",
-  "LINK/USDT",
-  "MANA/USDT",
-  "SOL/USDT",
-  "MATIC/USDT",
-  "THETA/USDT",
-  "HNT/USDT",
-  "DOT/USDT",
-  "YFI/USDT",
-  "LUNA/USDT",
-  "UNI/USDT",
-  "AVAX/USDT",
-  "AAVE/USDT",
-  "QNT/USDT",
-  "AXS/USDT",
-  "FIL/USDT"
+  // "BNB/USDT",
+  "BTC/USDT"
+  // 'USDC/USDT',
+  // 'ALGO/USDT',
+  // "ETH/USDT",
+  // "BNB/USDT",
+  // "BTC/USDT",
+  // "XRP/USDT",
+  // "ADA/USDT",
+  // "TRX/USDT",
+  // "BAT/USDT",
+  // "LINK/USDT",
+  // "MANA/USDT",
+  // "SOL/USDT",
+  // "MATIC/USDT",
+  // "THETA/USDT",
+  // "HNT/USDT",
+  // "DOT/USDT",
+  // "YFI/USDT",
+  // "LUNA/USDT",
+  // "UNI/USDT",
+  // "AVAX/USDT",
+  // "AAVE/USDT",
+  // "QNT/USDT",
+  // "AXS/USDT",
+  // "FIL/USDT"
```

Comment:
It seems three arrays above have quite similar items. You can use spread operator to avoid a lot of copy and paste.

```
const symbols = [A, B, C];
const extraSymbols = [...symbols, D, E]
// so you can get extraSymbols as [A, B, C, D, E]
```